buildSrc/src/main/groovy/org/elasticsearch/gradle/test/DistroTestPlugin.java

```diff
@@ -178,21 +212,30 @@ private void configureCopyPackagingTask(Project project) {
                     upgradeFromVersion = upgradeVersion.toString();
                 }
                 TaskInputs inputs = t.getInputs();
-                inputs.property("version", VersionProperties.getElasticsearch());
                 inputs.property("upgrade_from_version", upgradeFromVersion);
                 // TODO: this is serializable, need to think how to represent this as an input
                 //inputs.property("bwc_versions", bwcVersions);
```

Comment:
We can track `BwcVersions` by breaking it up into it's constituent parts. Alternatively, we could track the input to `BwcVersions` _itself_ which is simply `Version.java`

Reply:
I'll do that as a followup since this already existed from my previous PR.